### ISSUE TITLE
[Bug] Syntax autocomplete for invalid "aria-describedat" attribute

### ISSUE THREAD

**Issue by rahimabdi** [microsoft/monaco-editor#4164]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Actual (Problematic) Behavior

I notice that aria-describedat is offered as an autocomplete option for elements although it’s not a valid ARIA attribute (and never was formally included in the ARIA specification). Also, aria-describedat may be conflated with the valid aria-describedby property so perhaps removing it altogether would be best?

### Expected Behavior

Only `aria-describedby` should be a valid autocomplete option.

**Comment by hediet** [comment]
Can you please provide a monaco editor playground link? And a link to back up your finding for future reference. Thanks!

**Comment by hediet** [comment]
Cannot reproduce:

![chrome_4lZrp2jPkB](https://github.com/microsoft/monaco-editor/assets/2931520/db9a776d-c186-4eed-8f84-96f5a74375bb)
